specs/src/specs/data_structures.md

```diff
@@ -213,7 +213,7 @@ A proof for a leaf in a [binary Merkle tree](#binary-merkle-tree), as per Sectio
 
 ### Namespace Merkle Tree
 
-[Shares](./shares.md) in Celestia are associated with a provided _namespace_. The Namespace Merkle Tree (NMT) is a variation of the [Merkle Interval Tree](https://eprint.iacr.org/2018/642), which is itself an extension of the [Merkle Sum Tree](https://bitcointalk.org/index.php?topic=845978.0). It allows for compact proofs around the inclusion or exclusion of shares with particular namespace IDs.
+[Shares](./shares.md) in Celestia are associated with a provided _namespace_. The Namespace Merkle Tree (NMT) is a variation of the [Merkle Interval Tree](https://eprint.iacr.org/2018/642), which is itself an extension of the <!-- markdown-link-check-disable -->[Merkle Sum Tree](https://bitcointalk.org/index.php?topic=845978.0)<!-- markdown-link-check-enable -->. It allows for compact proofs around the inclusion or exclusion of shares with particular namespace IDs.
```

Comment:
proposal to extract the disable / enable to separate lines to make the content more readable.

```suggestion

<!-- markdown-link-check-disable -->
[Shares](./shares.md) in Celestia are associated with a provided _namespace_. The Namespace Merkle Tree (NMT) is a variation of the [Merkle Interval Tree](https://eprint.iacr.org/2018/642), which is itself an extension of the [Merkle Sum Tree](https://bitcointalk.org/index.php?topic=845978.0). It allows for compact proofs around the inclusion or exclusion of shares with particular namespace IDs.
<!-- markdown-link-check-enable -->
```

Reply:
Also, it can be helpful for maintainers to describe why a linter rule is disabled. So something like this may help:

https://github.com/celestiaorg/celestia-app/blob/60f932802e30c31c2f9dfa810bce0e74902cf716/specs/src/specs/data_structures.md?plain=1#L161-L162